exec/java-exec/src/main/java/org/apache/drill/exec/physical/rowSet/RowSet.java

```diff
@@ -15,7 +15,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.drill.test.rowSet;
+package org.apache.drill.exec.physical.rowSet;
```

Comment:
1. Remove public: `public interface SingleRowSet extends RowSet {`
2. Please try to fix all java doc errors.

Reply:
Thanks, done.